terms.html

```diff
@@ -5,12 +5,20 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made, by a specific <a>entity</a>, about a <a>resource</a>.
+This <a>resource</a> is called the <a>subject</a> of the <a>claim</a>.
+This digital representation usually consists of an (optional) identifier (a URL)
```

Comment:
This spec [requires identifiers to be URLs](https://www.w3.org/TR/vc-data-model-2.0/#identifiers) (the WHATWG version of URLs that includes URIs, etc.), so I believe we don't need to say "typically, ".

Reply:
@dlongley -- My thinking for the "typically" was that we allowed blank nodes for subjects, but I think you're right, that dropping the parenthetical is probably best. I'm updating my change request.

Reply:
I'll get rid of the entire sentence since, agreeing with @msporny in other comments, that should not be part of the definition, but properly specified in the body of the standard.